packages/gatsby-theme-docs/gatsby-browser.js

```diff
@@ -5,17 +5,17 @@
  * See: https://www.gatsbyjs.org/docs/browser-apis/
  */
 
+import React from 'react';
+import Prism from 'prism-react-renderer/prism';
+import { CacheProvider } from '@emotion/react';
+import { docsCache } from './utils/create-emotion-cache';
 import '@fontsource/roboto/latin-400.css';
 import '@fontsource/roboto/latin-500.css';
 import '@fontsource/roboto/latin-700.css';
 import '@fontsource/roboto-mono/latin-400.css';
 import '@fontsource/roboto-mono/latin-500.css';
 import '@fontsource/roboto-mono/latin-700.css';
 import './globals.css';
```

Comment:
some people said moving CSS imports to the end of the imports resolves some performance issues.